src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_LOTTERY_NUM = 100;
+    private Random random = new Random();
+
+    public Ball getRandomBall() {
+        int randomNumber = random.nextInt(MAX_LOTTERY_NUM);
+        ColorSupplier randomColor = new ColorSupplier();
```

Comment:
Move ColorSupplier to class fields. Don't create new ColorSupplier every time this method is invoked